### PR TITLE
CRYPTO-41:Update user guide according to the renamed classes and API …

### PR DESCRIPTION
**Pull request by kexianda**
[CRYPTO-41](https://issues.apache.org/jira/browse/CRYPTO-41)
class names have been changed.  change userguide according to the API changes

### CODE REVIEW

**Comment by kexianda** [comment]
add "Prerequisites" & "Using Commons Crypto in  Maven" in the userguide

File: src/site/xdoc/userguide.xml [hunk 1]
```diff
@@ -60,6 +60,21 @@
 
       <subsection name ="Usage">
         <ol style="list-style-type: decimal">
+          <h4>Prerequisites</h4>
+          <p>
+            Commons Crypto relies on standard JDK 7 (or above) and OpenSSL 1.0.1c (or above) for production
```
Comment (winningsix) [inline src/site/xdoc/userguide.xml]: I think 1.6 also works at this point.

File: src/site/xdoc/userguide.xml [hunk 2]
```diff
@@ -104,7 +119,7 @@
               byte[] output = new byte[1024];<br/>
               int outputOffset = 0;<br/>
               //Initializes the cipher with ENCRYPT_MODE,key and iv.<br/>
```
Comment (winningsix) [inline src/site/xdoc/userguide.xml]: Not related to this change. One space needed before the word "key"
```
//Initializes the cipher with ENCRYPT_MODE, key and iv
```

File: src/site/xdoc/userguide.xml (hunk 1 above)
Comment (kexianda) [inline src/site/xdoc/userguide.xml]: Hi @winningsix, thanks for you review.  JDK version was changed by haifeng  from 6 to 7 in this [commit](https://github.com/apache/commons-crypto/commit/32f789dbff523117223532ac8adfd5cacd1d350a)

File: src/site/xdoc/userguide.xml (hunk 2 above)
Comment (kexianda) [inline src/site/xdoc/userguide.xml]: :-)  fixed.  Thanks.

**Comment by winningsix** [comment]
LGTM

**Pull request closed without merging**